Build failure with GHC HEAD

```
System/Metrics.hs:448:16: error:
    • Constructor ‘Stats.GCStats’ does not have the required strict field(s): mblocksAllocated
    • In the expression:
        Stats.GCStats
          {bytesAllocated = 0, numGcs = 0, maxBytesUsed = 0,
           numByteUsageSamples = 0, cumulativeBytesUsed = 0, bytesCopied = 0,
           currentBytesUsed = 0, currentBytesSlop = 0, maxBytesSlop = 0,
           peakMegabytesAllocated = 0, mutatorCpuSeconds = 0,
           mutatorWallSeconds = 0, gcCpuSeconds = 0, gcWallSeconds = 0,
           cpuSeconds = 0, wallSeconds = 0, parTotBytesCopied = 0,
           parMaxBytesCopied = 0}
      In an equation for ‘emptyGCStats’:
          emptyGCStats
            = Stats.GCStats
                {bytesAllocated = 0, numGcs = 0, maxBytesUsed = 0,
                 numByteUsageSamples = 0, cumulativeBytesUsed = 0, bytesCopied = 0,
                 currentBytesUsed = 0, currentBytesSlop = 0, maxBytesSlop = 0,
                 peakMegabytesAllocated = 0, mutatorCpuSeconds = 0,
                 mutatorWallSeconds = 0, gcCpuSeconds = 0, gcWallSeconds = 0,
                 cpuSeconds = 0, wallSeconds = 0, parTotBytesCopied = 0,
                 parMaxBytesCopied = 0}
```

--- a/System/Metrics.hs
+++ b/System/Metrics.hs
@@ -445,6 +445,29 @@ getGcStats = do
 
 -- | Empty GC statistics, as if the application hasn't started yet.
 emptyGCStats :: Stats.GCStats
+#if MIN_VERSION_base(4,10,0)
+emptyGCStats = Stats.GCStats
+    { bytesAllocated         = 0
+    , numGcs                 = 0
+    , maxBytesUsed           = 0
+    , numByteUsageSamples    = 0
+    , cumulativeBytesUsed    = 0
+    , bytesCopied            = 0
+    , currentBytesUsed       = 0
+    , currentBytesSlop       = 0
+    , maxBytesSlop           = 0
+    , peakMegabytesAllocated = 0
+    , mutatorCpuSeconds      = 0
+    , mutatorWallSeconds     = 0
+    , gcCpuSeconds           = 0
+    , gcWallSeconds          = 0
+    , cpuSeconds             = 0
+    , wallSeconds            = 0
+    , parTotBytesCopied      = 0
+    , parMaxBytesCopied      = 0
+    , mblocksAllocated       = 0
+    }
+#else
 emptyGCStats = Stats.GCStats
     { bytesAllocated         = 0
     , numGcs                 = 0
@@ -465,6 +488,7 @@ emptyGCStats = Stats.GCStats
     , parTotBytesCopied      = 0
     , parMaxBytesCopied      = 0
     }
+#endif
 #else
 getGcStats = Stats.getGCStats
 #endif

--- a/System/Metrics.hs
+++ b/System/Metrics.hs
@@ -445,7 +445,7 @@ getGcStats = do
 
 -- | Empty GC statistics, as if the application hasn't started yet.
 emptyGCStats :: Stats.GCStats
-#if MIN_VERSION_base(4,10,0)
+# if MIN_VERSION_base(4,10,0)
 emptyGCStats = Stats.GCStats
     { bytesAllocated         = 0
     , numGcs                 = 0
@@ -467,7 +467,7 @@ emptyGCStats = Stats.GCStats
     , parMaxBytesCopied      = 0
     , mblocksAllocated       = 0
     }
-#else
+# else
 emptyGCStats = Stats.GCStats
     { bytesAllocated         = 0
     , numGcs                 = 0
@@ -488,7 +488,7 @@ emptyGCStats = Stats.GCStats
     , parTotBytesCopied      = 0
     , parMaxBytesCopied      = 0
     }
-#endif
+# endif
 #else
 getGcStats = Stats.getGCStats
 #endif

--- a/ekg-core.cabal
+++ b/ekg-core.cabal
@@ -31,7 +31,7 @@ library
 
   build-depends:
     ghc-prim < 0.6,
-    base >= 4.5 && < 4.10,
+    base >= 4.5 && < 4.11,
     containers >= 0.5 && < 0.6,
     text < 1.3,
     unordered-containers < 0.3

